stable/gcpdisk-csi-driver/templates/csi-gcpdisk-controller.yaml

```diff
@@ -0,0 +1,116 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-controller
+  namespace: {{ .Release.Namespace }}
+{{ include "gcpdisk.labels" . | indent 2 }}
+spec:
+  serviceName: "csi-gce-pd"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gcp-compute-persistent-disk-csi-driver
+  template:
+    metadata:
+{{ include "gcpdisk.labels" . | indent 6 }}
+        app: gcp-compute-persistent-disk-csi-driver
+    spec:
+      # Host network must be used for interaction with Workload Identity in GKE
+      # since it replaces GCE Metadata Server with GKE Metadata Server. Remove
+      # this requirement when issue is resolved and before any exposure of
+      # metrics ports
+      hostNetwork: true
+      serviceAccountName: csi-gce-pd-controller-sa
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      priorityClassName: system-cluster-critical
+      containers:
+        - name: csi-provisioner
+          image: quay.io/k8scsi/csi-provisioner:v1.4.0
+          args:
+            - --csi-address=$(ADDRESS)
+            - --timeout=30s
+            - --retry-interval-start=1s
+            - --retry-interval-max=5m
+            - --worker-threads=100
+            {{- if .Values.enableTopologyZone }}
+            - --feature-gates=Topology=true
+            {{- end}}
+            - --enable-leader-election
+            - --leader-election-type=leases
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          imagePullPolicy: Always
+          volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+        - name: cluster-driver-registrar
+          image: quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1
```

Comment:
we should expose this versions and tags :/

Reply:
then we should do it for azure as well
we can do it afterwords I would say.
as simply changing these tags does not mean that it will work.

Reply:
Yes we should do it there too, so we can bump versions without changing the chart that often

Reply:
sure